packages/calypso-e2e/src/lib/components/domain-search-component.ts

```diff
@@ -81,6 +81,8 @@ export class DomainSearchComponent {
 
 		await target.click();
 
+		await this.page.getByRole( 'button' ).filter( { hasText: 'Continue' } ).waitFor();
+
```

Comment:
@DustyReagan I am adding this line since there seems to be a delay the UI button that needs to be clicked.